package.json

```diff
@@ -86,6 +86,8 @@
     "@xkeys-lib/core@npm:3.3.0": "patch:@xkeys-lib/core@npm%3A3.3.0#~/.yarn/patches/@xkeys-lib-core-npm-3.3.0-1a65b6423b.patch",
     "unix-dgram": "npm:empty-npm-package@1.0.0",
     "bare-fs": "npm:empty-npm-package@1.0.0",
-    "bare-path": "npm:empty-npm-package@1.0.0"
+    "bare-path": "npm:empty-npm-package@1.0.0",
+    "@sentry/node-core@npm:10.17.0": "patch:@sentry/node-core@npm%3A10.22.0#~/.yarn/patches/@sentry-node-core-npm-10.22.0-e586fad107.patch",
+    "@sentry/node-core@npm:10.22.0": "patch:@sentry/node-core@npm%3A10.22.0#~/.yarn/patches/@sentry-node-core-npm-10.22.0-e586fad107.patch"
```

Comment:
These overrides worry me, as sentry is one of the dependencies that updates most frequently, and I will forget to update this patch, which result in it silently being dropped without me noticing.

I don't have a good solution to this though.. Might be a case of needing to fight webpack config?  
Just thinking out loud of some things I might try; Perhaps webpack can do similar mangling of a dependency as it bundles it?

Reply:
Working on this, but if all else fails, what do you think about just running main.js through `sed` to replace the offending string? (It would go in package.mts right before the launch check.)